video/external/src/decoder/openh264_sys.rs

```diff
@@ -1359,11 +1359,11 @@ pub struct OpenH264 {
         Result<unsafe extern "C" fn(pVersion: *mut OpenH264Version), ::libloading::Error>,
 }
 impl OpenH264 {
-    pub unsafe fn new<P>(path: P) -> Result<Self, ::libloading::Error>
+    pub unsafe fn new<P>(filename: P) -> Result<Self, ::libloading::Error>
     where
         P: AsRef<::std::ffi::OsStr>,
```

Comment:
Is it just me, or does this feel inconsistent now, with the `P` not following the rename...? :thinking:

Reply:
Huh, you're right, but that's how libloading does it:
```rust
pub unsafe fn new<P: AsRef<OsStr>>(filename: P) -> Result<Library, Error> {
```

I could change that to `F`, but in reality it might be either path or filename, so I don't know what's best here. My goal was to match the signature with libloading in order not to suggest that we are supporting paths only.

Reply:
Being consistent with `libloading` is fine then.